WordPress/Classes/ViewRelated/Blog/SharingDetailViewController.m

```diff
@@ -119,11 +139,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     if (indexPath.section == 0) {
         cell = [self switchTableViewCell];
 
-    } else if (indexPath.section == 1 && [self.publicizeConnection isBroken]) {
-        cell.textLabel.text = NSLocalizedString(@"Reconnect", @"Verb. Text label. Tapping attempts to reconnect a third-party sharing service to the user's blog.");
-        [WPStyleGuide configureTableViewActionCell:cell];
-        cell.textLabel.textAlignment = NSTextAlignmentCenter;
-        cell.textLabel.textColor = [WPStyleGuide jazzyOrange];
+    } else if (indexPath.section == 1) {
+        if ([self.publicizeConnection isBroken]) {
```

Comment:
I would also suggest extracting each logical branch to a separate method, if possible. That would make this code easier to read, to begin with. Also, there seems to be an issue with this title (more in another comment), so making this code simpler to read might have the extra advantage of fixing that :)